INSTALL.rst

```diff
@@ -1,9 +1,30 @@
-============
-Installation
-============
+..
+    This file is part of INSPIRE.
+    Copyright (C) 2017 CERN.
+
+    INSPIRE is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    INSPIRE is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+
+    In applying this licence, CERN does not waive the privileges and immunities
+    granted to it by virtue of its status as an Intergovernmental Organization
+    or submit itself to any jurisdiction.
+
+
+Native Install (CentOS - MacOS)
```

Comment:
Not sure if it needs to be done in this PR, but adding in the title both systems might be misleading.

For example, in the first section you use `yum` which is not available in MacOS. If there is nothing specific of MacOS you can remove it from the title and later on we can add some specific section with Ubuntu or MacOS if needed.